challenges/challenge-1/challenge-1.css

```diff
@@ -2,7 +2,7 @@
 
 strong {
   font-size: 1.2em;
-  font-family: Cambria, Cochin, Georgia, Times, "Times New Roman", serif;
```

Comment:
Unfortunately, your formatting changed all of our double quotes to single quotes in the challenge-1.css file.